src/sdl2_wrapper.cpp

```diff
@@ -812,6 +812,21 @@ CAMLprim value resdl_SDL_SetWindowPosition(value vWin, value vX, value vY) {
   CAMLreturn(Val_unit);
 }
 
+CAMLprim value resdl_SDL_GetWindowPosition(value vWin) {
+  CAMLparam1(vWin);
+  int x, y;
+  SDL_Window *win = (SDL_Window *)vWin;
+  SDL_GetWindowPosition(win, &x, &y);
+
+  CAMLlocal1(ret);
+  ret = caml_alloc(2, 0);
+
+  Store_field(ret, 0, x);
```

Comment:
Does this need to be `Store_field(ret, 0, Val_int(x))`? I recall sometimes that there would be issues with this - see `resdl_SDL_GetWindowSize`/`resdl_SDL_GetDrawableSize`?